grails-app/conf/data/mapping.json

```diff
@@ -70,17 +70,16 @@
         "organisationId": {
           "type" : "keyword"
         },
-        "orgIdSvcProvider": {
-          "type" : "keyword",
-          "copy_to": ["organisationId"]
-        },
         "organisationName": {
           "type" : "text",
           "copy_to": ["organisationFacet", "organisationSort"]
         },
         "organisationFacet": {
           "type": "keyword"
         },
+        "linkedOrganisationFacet": {
```

Comment:
is this required?

Reply:
I've actually just got the requirements for how the organisation facet should work in MERIT this morning - we were originally going to have 2 organisation facets with slightly different behaviour (which is what this was going to be used for) but I think we no longer need it so I've removed it.